libtcmu_common.h

```diff
@@ -39,6 +39,9 @@ struct tcmulib_cmd;
 #define CFGFS_ROOT "/sys/kernel/config/target"
 #define CFGFS_CORE CFGFS_ROOT"/core"
 
+#define VPD_MAX_UNMAP_LBA_COUNT            1048576
```

Comment:
Was this the ESX default? Could you add what version of ESX and vmfs it was?

I think in ESX 6.5 with vmfs 6, esx will use the value returned to it, so instead of making the ceph object size match esx's limit, in future patches we can report the ceph object size as a handler limit like we do with max sectors and ESX will use that.